sass/components/_tutorial.scss

```diff
@@ -78,7 +78,7 @@ $tip-index: 1000;
         .arrow {
             top: 34px;
             left: -10px;
-            border-right-color: $white;
```

Comment:
Thanks @AGMETEOR , This solves the problem. However, I think it is worth checking if it is only this `$white ` variable or if all the other included variables in this file are not working.

Reply:
Looking over this file quickly, seems like there are other instances of `$white` that should be replaced. I think the instances of `$black` are all for shadows and can stay as they are.

Reply:
@neallred  I only replaced those instances where I could see the color of the arrow being white on a dark theme. Is it safe to replace all those other `$white` variables with the `--center-channel-bg`

Reply:
I haven't checked. If you checked the various tips on dark theme and they look good with the changes you have now, I'd say keep it as you have it.